components/inputs/sass/textarea.scss

```diff
@@ -1,14 +1,14 @@
-@import "../../colors/colors.scss";
-@import "./text.scss";
+@use "../../colors/colors.scss";
+@use "./text.scss";
 
 @mixin d2l-input-textarea() {
-	@include d2l-input-text(
+	overflow: auto;
+	@include text.d2l-input-text(
 		$include-pseudo-classes: false,
 		$line-height: normal,
 		$padding: 0.5rem 0.75rem,
 		$padding-focus: calc(0.5rem - 1px) calc(0.75rem - 1px)
 	);
-	overflow: auto;
```

Comment:
Having rules declared after an `@include` is also deprecated. Moving it up to the top has no effect on the CSS.